WordPress/Classes/ViewRelated/Tools/SettingsListPickerViewController.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import WordPressShared
+
+
+
+/// SettingsListPicker will render a list of options, and will allow the user to select one from the list.
+///
+class SettingsListPickerViewController : UITableViewController
+{
+    /// Header Strings to be applied over the diferent sections
+    ///
+    var headers : [String]?
+
+    /// Footer Strings to be applied over the diferent sections
+    ///
+    var footers : [String]?
+    
+    /// Titles to be rendered
+    ///
+    var titles : [[String]]?
+
+    /// Row Subtitles. Should contain the exact same number as titles
+    ///
+    var subtitles : [[String]]?
```

Comment:
I will prefer to make them private and have the configure method for all them that will make it more explicit and also avoid that people go and change it after viewWillAppear is triggered.